src/lib/battery/module.yaml

```
@@ -105,13 +105,13 @@ parameters:
             description:
                 short: Battery ${i} monitoring source.
                 long: |
-                    This parameter controls the source of battery data. The value 'Power Module'
+                    This parameter controls the source of battery data. The value 'Power Module / Analog'
                     means that measurements are expected to come from either analog (ADC) inputs
                     or an I2C power monitor (e.g. INA226). Analog inputs are voltage and current
                     measurements read from the board's ADC channels, typically from an onboard
                     voltage divider and current shunt, or an external analog power module.
                     I2C power monitors are digital sensors on the I2C bus.
-                    If the value is set to 'External' then the system expects to receive mavlink
+                    If the value is set to 'External' then the system expects to receive MAVLink
                     battery status messages or the battery data is published by an external driver.
```

@dakejahl IMO this should be sufficient to capture that CAN is external. I'm going to commit this and approve.
Happy for you to then merge at will.

```suggestion
                    or CAN battery status messages, or the battery data is published by an external driver.
```